pymc3/distributions/continuous.py

```diff
@@ -2478,7 +2481,17 @@ def logcdf(self, value):
         """
         alpha = self.alpha
         beta = self.beta
-        return bound(tt.log(tt.gammainc(alpha, beta * value)), value >= 0, alpha > 0, beta > 0)
+        # To avoid issue with #4340
```

Comment:
Better to say what the actual issue is rather than point elsewhere.
Better still to explain this in the commit message.

Reply:
Agree, will do.